tests/drivers/build_all/dac/testcase.yaml

```
@@ -1,8 +1,28 @@
 common:
   build_only: true
-  tags: drivers
+  tags: drivers dac
 tests:
-  drivers.dac.build:
-    min_ram: 32
-    depends_on: dac spi
-    platform_exclude: serpente
+  drivers.dac.spi.build:
+    platform_allow: native_posix
+    tags: dac_dacx0508
+    extra_args: "CONFIG_SPI=y CONFIG_DAC_DACX0508=y"
+  drivers.dac.i2c.build:
+    platform_allow: native_posix
+    tags: dac_dacx3608
```

```suggestion
    tags: dac_dacx3608 dac_mcp4725
```

Or did you mean to split them in two test cases?